lib/src/geometry/polygon.dart

```diff
@@ -0,0 +1,156 @@
+import 'dart:math';
+import 'dart:ui';
+
+import '../../components.dart';
+import '../extensions/rect.dart';
+import '../extensions/vector2.dart';
+import 'shape.dart';
+
+class Polygon extends Shape {
+  final List<Vector2> definition;
```

Comment:
how about vertexes? or maybe relativeVertexes? I feel definition is a bit unclear (thought I get what you mean)

Reply:
or vertices not sure what is the right plural